js_functions/task3.js

```diff
@@ -0,0 +1,6 @@
+function f1(num) {
```

Comment:
function name should represent the action, use more informative name like `countDown`